app/uk/gov/hmrc/fileupload/controllers/envelope.scala

```diff
@@ -62,22 +62,35 @@ object EnvelopeReport {
 case class CreateEnvelopeRequest(callbackUrl: Option[String] = None,
                                  expiryDate: Option[DateTime] = None,
                                  metadata: Option[JsObject] = None,
-                                 constraints: Option[EnvelopeConstraints] = Some(defaultConstraints))
+                                 constraints: Option[EnvelopeConstraintsUserSetting] = Some(EnvelopeConstraintsUserSetting()))
 
+case class EnvelopeConstraintsUserSetting(maxItems: Option[Int] = Some(defaultMaxItems),
+                                          maxSize: Option[Long] = Some(defaultMaxSize),
+                                          maxSizePerItem: Option[Long] = Some(defaultMaxSizePerItem),
+                                          contentTypes: Option[List[ContentTypes]] = Some(defaultContentTypes))
```

Comment:
We are filling defaults in two places. 
at case class EnvelopeConstraintsUserSetting leave the default as None instead of defaults